src/components/CreatePost/CreatePost.tsx

```diff
@@ -49,64 +54,222 @@ const CreatePost = () => {
     }));
   };
 
-  const firestore = useContext(FirestoreContext);
+  const { firestore, storage } = useContext(FirebaseContext);
+
+  const [title, setTitle] = useState<string>('');
+  const [description, setDescription] = useState<string>('');
+  const [selectedFiles, setSelectedFiles] = useState<FilesInterface[]>([]);
+  const [selectedImages, setSelectedImages] = useState<ImagesInterface[]>([]);
+  const [showForm, setShowForm] = useState<boolean>(true);
+  const [titleChars, setTitleChars] = useState<number>(maxTitleChars);
+  const [descrChars, setDescrChars] = useState<number>(maxDescrChars);
+
+  const fileInput = React.createRef<HTMLInputElement>();
+
+  const storeImages = async () => {
```

Comment:
Functions like these should be extracted to the firebase file like we discussed before.

Reply:
It would be nice if we could just do `firebase.storeImages(selectedFiles)`